tools/cpplint.py

```diff
@@ -558,10 +567,11 @@ def IncrementErrorCount(self, category):
 
   def PrintErrorCounts(self):
     """Print a summary of errors by category, and the total."""
-    for category, count in self.errors_by_category.iteritems():
-      sys.stderr.write('Category \'%s\' errors found: %d\n' %
-                       (category, count))
-    sys.stderr.write('Total errors found: %d\n' % self.error_count)
+    if not _cpplint_state.output_format == 'tap':
```

Comment:
Any specific reason to use this instead of `if _cpplint_state.output_format != 'tap':`?